xarray/test/test_dataset.py

```diff
@@ -685,6 +686,23 @@ def test_coords_merge_mismatched_shape(self):
         actual = orig_coords.merge(other_coords)
         self.assertDatasetIdentical(expected, actual)
 
+    def test_data_vars_properties(self):
```

Comment:
It looks like we were missing tests for `data_vars` previously, which I noticed when I renamed the formatting function and an error appeared in the docs build. Oops!